### DISCUSSION TITLE
有点不能理解为什么丰饶命途战斗时要按3和r

### DISCUSSION THREAD

**Discussion by Eric-Joker** [CHNZYX/Auto_Simulated_Universe#143] [General]
本来生存能力就不太够，一直按3和r不按自动战斗的逻辑走生存能力更差了。

**Comment by Eric-Joker** [comment]
而且3也不一定是丰饶角色啊（

**Comment by Eric-Joker** [comment]
emm，想起了自动战斗逻辑貌似雀食不会及时的触发回血技能。但是如果一直按的话更多的情况是浪费充能。如果可能的话血量监测？

**Comment by CHNZYX** [comment]
为了触发露珠。一般来说选丰饶即使有大就开，生存能力也会很强，所以权衡效率和生存后选择了有大就开。如果你的生存能力不够，可以把奶放到其他位置。
